frontend/src/views/Resources.vue

```diff
@@ -7,7 +7,7 @@
         class="leaderboard-header">
       <h1 class="resources-heading">Contributing to Open Source</h1>
       <p>To get off to a great start, here are some resources that you may find of use. If you have any questions, please don't hesitate to talk to others at the event or reach out to any of the organisers. We are happy to help!</p>
-      <p>Contributing to Open Source is actually quite easy once you find the right project and know how to use GitHub. For an awesome guide, checkout GitHub's own excellent guide <a 
+      <p>Contributing to Open Source is actually quite easy once you find the right project and know how to use GitHub. For an awesome guide, checkout GitHub's own excellent guide!<a 
```

Comment:
Shouldn't be a ! here because it's not the end of the sentence. The sentence is: 
For an awesome guide, checkout GitHub's own excellent guide <a>here</a>